_locales/en_US/messages.json

```diff
@@ -665,19 +665,19 @@
 		"message": "Default signers rules"
 	},
 	"treeviewSigners.treecol.domain": {
-		"message": "Domain"
+		"message": "Domain:"
```

Comment:
My guess you want to add a `:` for the rules adding dialog?
Unfortunately the `treeviewSigners.treecol.<...>` labels are reused multiple times. The main use is the column label of the table views, there a `:` at the end doesn't make sense.
Don't know if it would be worth it duplicating them just for the added `:`.